src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -939,8 +939,8 @@ def getWorkloadArguments():
                      "ScramArch": {"default": "slc5_amd64_gcc462", "optional": False},
                      "GlobalTag": {"null": True},
                      "GlobalTagConnect": {"null": True},
-                     "ProcessingVersion": {"default": 1, "type": int},
-                     "ProcessingString": {"null": True},
+                     "ProcessingVersion": {"default": 1, "type": int, "assign_optional": False},
+                     "ProcessingString": {"default": "None", "validate": procstring, "assign_optional": False},
```

Comment:
I suggest we default it to the same default value as AcquisitionEra, so "FAKE". Easier to keep track of what to expect on the assignment side.